File: Python/4A-Watermelon.py
import sys

def main():
    # Read the input weight
    w = int(sys.stdin.read().strip())
    
    # Check if the weight is an even number greater than 2
    if w > 2 and w % 2 == 0:
        print("YES")
    else:
        print("NO")

if __name__ == "__main__":
    main()

File: CPP/4A-Watermelon.cpp
#include <iostream>

using namespace std;

int main() {
    // Fast I/O
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    
    int w;
    cin >> w;
    
    // Check if the weight is an even number greater than 2
    if (w > 2 && w % 2 == 0) {
        cout << "YES\n";
    } else {
        cout << "NO\n";
    }
    
    return 0;
}
